src/components/Search/SearchPageHeader.tsx

```diff
@@ -107,21 +110,18 @@ function SearchPageHeader({query, selectedItems = {}, hash, clearSelectedItems}:
             });
         }
 
-        return (
-            <ButtonWithDropdownMenu
-                onPress={() => null}
-                shouldAlwaysShowDropdownMenu
-                pressOnEnter
-                buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
-                customText={translate('workspace.common.selected', {selectedNumber: selectedItemsKeys.length})}
-                options={options}
-                isSplitButton={false}
-                isDisabled={isOffline}
-            />
-        );
-    }, [clearSelectedItems, hash, isOffline, selectedItems, styles.colorMuted, styles.fontWeightNormal, theme.icon, translate]);
+        return options;
+    }, [clearSelectedItems, hash, selectedItems, selectedItemsKeys, styles, theme, translate]);
 
     if (isSmallScreenWidth) {
+        if (isMobileSelectionModeActive) {
+            return (
+                <SearchSelectedNarrow
+                    options={headerButtonsOptions}
```

Comment:
For some reason the options are not showing up for me, is the same happening for anyone else?

Reply:
It should works right now